website/docs/components/toast/partials/guidelines/guidelines.md

```diff
@@ -51,8 +50,35 @@ Use color logically.
 - **Highlight** use interchangeably with `neutral` when more prominence is needed. Use it sparingly.
 - **Success** to indicate a successful action was completed.
 - **Warning** to indicate a successful action was completed but may have triggered a related issue. Provide guidance and actions if possible.
+
 - **Critical** to indicate error or critical issues resulting from a failed action.
 
+## Critical Toast vs Critical Alert
+
+### When to use a Toast
+
+Use the Toast to provide non-intrusive feedback to users about the failure of an ongoing task or request. For example, a failure while deleting a cluster.
+
+<Hds::Toast @color="critical" @onDismiss={{this.noop}} as |T|>
+    <T.Title>Module error</T.Title>
+    <T.Description>This module encountered an error during publishing. You may need to republish.</T.Description>
+  </Hds::Toast>
+
+Use the Toast to communicate error messages that are not caused by the user. For example, an unsuccessful Vault cluster creation due to a failure while validating the deployment.
+
+<Hds::Toast @color="critical" @onDismiss={{this.noop}} as |T|>
+    <T.Title>Vault cluster update failed</T.Title>
+    <T.Description>**test-cluster_1** configuration update failed.</T.Description>
+  </Hds::Toast>
+
+<!-- Do not set critical Toast to dismiss automatically. -->
```

Comment:
?

Reply:
Sometimes I comment out paragraphs or part of an idea I need later on or haven't completed. I forgot to delete it. It should be gone now.